docs/content/v2024.2/explore/colocation.md

```diff
@@ -237,3 +237,132 @@ To set up xCluster for colocated tables, do the following:
 If new colocated tables are added to the same colocated database on both source and target universes with matching colocation IDs, then they are automatically included in replication.
 
 For information on how to set up xCluster for non-colocated tables, refer to [xCluster deployment](../../deploy/multi-dc/async-replication/).
+
+## Colocated tables with tablespaces
+
+Colocated tables can be placed in [tablespaces](../going-beyond-sql/tablespaces/) {{<tags/feature/ea>}}. When a colocated table is created in a tablespace, the colocation tablet is placed and replicated exclusively in the tablespace.
+
+During Early Access, by default colocation support for tablespaces is not enabled. To enable the feature, set the flag `ysql_enable_colocated_tables_with_tablespaces=true`.
+
+### Create a colocated table in a tablespace
+
+In a [colocated database](#databases), [tables](#tables) are created with colocation by default. To create a colocated table in a tablespace, use the following command:
+
+```sql
+CREATE TABLE <table_name> TABLESPACE <tablespace_name>;
+```
+
+Use the same syntax to create indexes and materialized views in a tablespace as follows:
+
+```sql
+CREATE INDEX <index_name> ON <table_name>(<column_name>) TABLESPACE <tablespace_name>;
+CREATE MATERIALIZED VIEW <view_name> TABLESPACE <tablespace_name> AS <query>;
+```
+
+To create a non-colocated table (in a colocated database) in a tablespace, use the following command:
+
+```sql
+CREATE TABLE <table_name> WITH (COLOCATION=FALSE) TABLESPACE <tablespace_name>;
+```
+
+### View tablespace and colocation properties
+
+To check the tablespace and colocation properties of a table, use the [\d](../../api/ysqlsh-meta-commands/#d-s-pattern-patterns) meta-command on the table as follows:
+
+```sql
+\d table_name;
+```
+
+You should see output similar to the following:
+
+```output
+                 Table "public.t"
+ Column |  Type   | Collation | Nullable | Default
+--------+---------+-----------+----------+---------
+ col    | integer |           |          |
+Tablespace: "us_east_1a_zone_tablespace"
+Colocation: true
+```
+
+### List all tablegroups and associated tablespaces
+
+To list all tablegroups and their associated tablespaces, use the following query:
+
+```sql
+SELECT * FROM pg_yb_tablegroup;
+```
+
+You should see output similar to the following:
+
+```output
+  oid  |     grpname      | grpowner | grptablespace | grpacl | grpoptions
+-------+------------------+----------+---------------+--------+------------
+ 16387 | default          |       10 |             0 |        |
+ 16391 | colocation_16384 |       10 |         16384 |        |
+(2 rows)
+```
```

Comment:
Please remove the oid column. The oid column shows up only in pg15, in 2024.2 it won't be there. Sorry this was a miss from my side, in the original doc.